crates/re_log_types/src/component_types/tensor.rs

```diff
@@ -761,60 +761,3 @@ fn test_arrow() {
     let tensors_out: Vec<Tensor> = TryIntoCollection::try_into_collection(array).unwrap();
     assert_eq!(tensors_in, tensors_out);
 }
-
-#[test]
-fn test_concat_and_slice() {
```

Comment:
This is basically just testing that `arrow2` and `arrow2-convert` do the correct thing with unions, which is now tested upstream since we've upstreamed the fixes.